packages/core/src/components/tag/stories/tag.react.stories.tsx

```diff
@@ -0,0 +1,99 @@
+import { Meta, StoryObj } from '@storybook/react'
+import React from 'react'
+
+import { AtomTag } from '@juntossomosmais/atomium/react'
+
+import { TagStoryArgs } from './tag.args'
+
+export default {
+  title: 'Components/Tag',
+  component: AtomTag,
+  ...TagStoryArgs,
+} as Meta
+
+const createTag = (args) => <AtomTag color={args.color}>{args.label}</AtomTag>
+
+export const Success: StoryObj = {
+  render: (args) => createTag(args),
+  args: {
+    type: 'success',
+    label: 'Promoção',
```

Comment:
(block) as this is a public project, is important follows writing all content using english

Reply:
@felipefialho done here https://github.com/juntossomosmais/atomium/commit/f387db24e1fdecf8d3b28df949f21e0d42d25d5d